common/src/main/java/uk/nhs/adaptors/common/util/FileUtil.java

```diff
@@ -5,11 +5,15 @@
 import java.io.FileNotFoundException;
 import java.io.InputStream;
 
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
 import org.apache.commons.io.IOUtils;
 
 import lombok.SneakyThrows;
 
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
Just a question here.  What is this line that has been added doing?

Reply:
it just makes this Util effectively working as a util class and not something that can be instantiated with "new FileUtil" class.  It is just making whatever we declare by name (....Util) to work as it was declared..

Reply:
ahh like static abstract class.. thanks